setup.py

```diff
@@ -47,8 +47,9 @@ def get_version(*file_paths):
     ],
     include_package_data=True,
     install_requires=[
-        "celery>=3.1.25,<4.0",
+        "celery>=3.1,<4.0",
```

Comment:
Specifying `3.1.25` here was upgrading the installed version of celery on my devstack, which I was not intending to upgrade for all of edx-platform.